spider.py

```diff
@@ -11,13 +11,14 @@
 
 def is_dir(url):
     """ Returns True if given url belongs to a directory, otherwise returns false """
-    return True if url[-1] == '/' else False
+    # could be return url[-1] == '/' as well
+    return url.endswith('/')
 
 
 def write_link(url):
     """ Appends links to a text file """
     with open('links.txt', 'a') as file:
-        file.write(url.lower() + "\n")
```

Comment:
used lower() for string comparison

Reply:
That's all good, just thought that if you ever wanted to go back to a url that you had saved, you may not be able to find the resource if the original url had uppercase characters.